python/lsst/ctrl/execute/slurmPlugin.py

```diff
@@ -246,25 +420,31 @@ def glideinsFromJobPressure(self):
                 condorq_bps = condorq_data[schedd_name]
                 if verbose:
                     print(len(condorq_bps))
-                    print(condorq_bps)
+                    # This can be extremely large
+                    # print(condorq_bps)
                 # disassemble the dictionary of dictionaries
                 for jid in list(condorq_bps.keys()):
                     job = condorq_bps[jid]
                     thisCores = job["RequestCpus"]
                     thisMemory = job["RequestMemory"]
+                    if thisCores > autoCPUs or thisMemory > autoCPUs * memoryPerCore:
```

Comment:
Same question about putting this in the condor_q constraint.

Reply:
It has been placed in the condor_q constraint.